python/lammps/mliap/__init__.py

```diff
@@ -31,5 +31,8 @@
                   "in undefined behavior.")
 else:
     from .loader import load_model, load_unified, activate_mliappy
-
+    try:
```

Comment:
Rather than depending on the package load failing and catching the exception, this should use the available introspection to see if the KOKKOS package is included in LAMMPS and only then try to load the modules.

Reply:
Do we have an example of this somewhere I can copy?

Reply:
There is an example in `unittest/python/python-pizza.py`. But after having a closer look at this and making some tests, I have to conclude that you have no alternative here. This code is run *before* you have access to an instance of the LAMMPS class and thus access to the loaded LAMMPS shared library and that does not give you access to the compiled in features. The cure would be worse than the disease. :-(